src/App.tsx

```diff
@@ -17,18 +52,28 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                status={status}
+                query={query}
+                setStatus={setStatus}
+                setQuery={setQuery}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {isLoading && <Loader />}
+              <TodoList
+                todos={filteredTodos}
+                selectedTodo={selectedTodo}
+                onSelectedTodo={setSelectedTodo}
+              />
```

Comment:
I think, we have a bug here
```suggestion
              {isLoading ? (
                <Loader />
              ) : (
                <TodoList
                  todos={filteredTodos}
                  selectedTodo={selectedTodo}
                  onSelectedTodo={setSelectedTodo}
               />
              )}
```
<img width="1247" alt="Знімок екрана 2024-10-15 о 14 36 31" src="https://github.com/user-attachments/assets/a400178f-65da-4004-8122-661191f8c1b3">